modules/openapi-generator/src/main/resources/rust-axum/models.mustache

```diff
@@ -644,28 +644,28 @@ pub struct {{{classname}}} {
         {{/pattern}}
         {{#maximum}}
             {{#minimum}}
-            range(min = {{minimum}}, max = {{maximum}}),
+            range(min = {{minimum}}{{{dataType}}}, max = {{maximum}}{{{dataType}}}),
             {{/minimum}}
             {{^minimum}}
-            range(max = {{maximum}}),
+            range(max = {{maximum}}{{{dataType}}}),
             {{/minimum}}
         {{/maximum}}
         {{#minimum}}
             {{^maximum}}
-            range(min = {{minimum}}),
+            range(min = {{minimum}}{{{dataType}}}),
             {{/maximum}}
         {{/minimum}}
         {{#maxItems}}
             {{#minItems}}
-            length(min = {{minItems}}, max = {{maxItems}}),
+            length(min = {{minItems}}u64, max = {{maxItems}}u64),
             {{/minItems}}
             {{^minItems}}
-            length(max = {{maxItems}}),
+            length(max = {{maxItems}}u64),
```

Comment:
imho, I don't think of any APIs need 2^64-1 items, even 2^32-1 items is impossible.

Reply:
This was done to guarantee that the value provided is compatible with [ValidateLength](https://docs.rs/validator/latest/validator/trait.ValidateLength.html). The trait is only implemented for u64, other types like u32, or even the more adequate usize do not work. This just serves as a clarification to the end user that only this integer type will work on that field, and enforces unsigned integers.

Reply:
IMHO:
1. Regarding to the [specs](https://swagger.io/docs/specification/v3_0/data-models/data-types/#numbers), integer should be in range of int64 instead. Specify maxItems as u64 like #19291 is invalid. Breaking specs of OpenAPI and also the Generator.
2. [validator](https://docs.rs/validator/latest/validator/index.html) is `generic`. Type is inferred. Specifying the type here sounds unreasonable to me.

Reply:
I have removed the length type, it's not needed.

Though, it's good to keep in mind that according to the [JSON Schema Specification](https://json-schema.org/draft/2020-12/json-schema-validation#name-validation-keywords-for-str), which OpenAPI follows, the value provided in `{min,max}Length` must be a non-negative number, this makes `u64` the adequate choice for the type, rather than `i64`, the `int64` equivalent in rust. This only breaks spec when dealing with integers matching `N >= (1<<63)`, as they do not fit in `int64`.